### ISSUE TITLE
Please add symlog for yscale in pyplot backend

### ISSUE THREAD

**Issue by gaomrx** [JuliaPlots/Plots.jl#1235]
Something similar to what [pyplot](https://matplotlib.org/gallery/pyplots/pyplot_scales.html#sphx-glr-gallery-pyplots-pyplot-scales-py) produces will be great. Alternatively, how can one plot the following with a y-log axis ` x = linspace(1,10, 10)
y = linspace(0, 1e5, 10)`

**Comment by mkborregaard** [comment]
You can't, because 0 can't be expressed in log space.

**Comment by gaomrx** [comment]
![image](https://user-images.githubusercontent.com/6380557/32741104-22369006-c873-11e7-9dde-a6a26f6c0268.png)
That was a bad example. How about restricting y to be in (0, Inf)?

**Comment by mkborregaard** [comment]
Sorry, I'm not sure I follow completely

**Comment by gaomrx** [comment]
If I restrict y to be in (0,Inf), then log(y) should be a value that can be plotted in log space right?

**Comment by mkborregaard** [comment]
yes I'd say so

**Comment by Omega-xyZac** [comment]
Would love to know how to do this!

**Comment by isentropic** [comment]
I can attempt to implement this, but please let me know more about the symlog and how to implement it mathematically. It'd be a good addition.

**Comment by Omega-xyZac** [comment]
Hi @isentropic, I think there should be enough to go off here https://discourse.julialang.org/t/symmetrical-log-plot/45709. It appears to be a natural extension of the log function to the negative numbers (and zero). There is also already an implementation from matplotlib that you might be able to go off. I haven't found where it is yet though.

**Comment by mkborregaard** [comment]
Should it maybe be changed here? https://github.com/JuliaPlots/RecipesPipeline.jl/blob/19dd30b3f68d29476448829141f42b2372fdd6ba/src/utils.jl#L193-L205

**Comment by Omega-xyZac** [comment]
Yes, that looks like an appropriate place. I don’t know a great deal about plots under the hood.

On 8 Feb 2021, at 10:00 pm, Michael Krabbe Borregaard <notifications@github.com> wrote:

﻿

Should it maybe be changed here? https://github.com/JuliaPlots/RecipesPipeline.jl/blob/19dd30b3f68d29476448829141f42b2372fdd6ba/src/utils.jl#L193-L205<https://urldefense.com/v3/__https://github.com/JuliaPlots/RecipesPipeline.jl/blob/19dd30b3f68d29476448829141f42b2372fdd6ba/src/utils.jl*L193-L205__;Iw!!NVzLfOphnbDXSw!VP0QcEpWXAHvqWUUhFygvEUXDDqH-oECSxCL7fPBmmpYa0pocBgT06zFgvdyA3OtSpHetr4C$>

**Comment by gvdeynde** [comment]
As far as I can see, symlog is not yet implemented in the pythonplot (nor any other backend). Is this something that can/will be done in the near future?

**Comment by BeastyBlacksmith** [comment]
Strongly depends on whether someone has a need for it and implements that

**Comment by gvdeynde** [comment]
I have a need but I'm not fluent enough in Julia to implement it. Still learning... Sorry.

**Comment by Seanlyons316** [comment]
Did anyone figure out how to resolve this? Can you call a wrapper from python for symlog?
